### PR TITLE
new icon: moleculer (original, plain)

### LINKED ISSUE #1300: [ICON REQUEST]: moleculer
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Progressive microservices framework for Node.js.

### Links and sources

Website -> https://moleculer.services/
Logos -> https://github.com/moleculerjs/branding/tree/master/logo

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kilian-paquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1300**

## Link to prove your SVG is correct and up-to-date.

https://moleculer.services/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -4946,6 +4946,31 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "moleculer",
+        "altnames": [],
```
Comment (Snailedlt) [inline devicon.json:4951]:
```suggestion
        "altnames": [
             "moleculerjs"
             "moleculer.js"
         ],
```

File: devicon.json [hunk 2]
```diff
@@ -4946,6 +4946,31 @@
...
+        "tags": [
+            "nodejs",
```
Comment (Snailedlt) [inline devicon.json:4953]:
```suggestion
            "nodejs",
            "javascript",
            "js",
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Some small changes. I also noticed that they have a few more logos, I'll make an issue so we can discuss further there

File: devicon.json [hunk 3]
```diff
@@ -4946,6 +4946,36 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "moleculer",
+        "altnames": [
+             "moleculerjs"
```
Comment (kilian-paquier) [inline devicon.json:4952]:
```suggestion
             "moleculerjs",
```

File: devicon.json [hunk 4]
```diff
@@ -4946,6 +4946,42 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "moleculer",
```
Comment (Snailedlt) [inline devicon.json:4950]: Please move this to the correct place so it's in alphabetical order. See [this relevant comment](https://github.com/devicons/devicon/pull/1289#issuecomment-1198001547) and [this issue](https://github.com/devicons/devicon/issues/1327) for more info

**Review by Snailedlt — CHANGES_REQUESTED** [review]

**Comment by kilian-paquier** [comment]
@Snailedlt Done. I also updated all my PRs to replace the new object at the right place 🙂

**Comment by Snailedlt** [comment]
> @Snailedlt Done. I also updated all my PRs to replace the new object at the right place 🙂

Great, you are so damn fast, lol!

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️ 
Thanks for your contribution! 🚀

**Pull request merged**